labs/lab-04/tasks/fibonacci/README.md

```diff
@@ -9,8 +9,29 @@ You will solve this exercise starting from the `fibonacci.asm` file located in t
 
 Calculate the Nth Fibonacci number, where N is given through the `eax` register.
 
->**NOTE**: The fibonacci series goes as follows : `0, 1, 1, 2, 3, ...` where each element `f[i] = f[i-2] + f[i-1]`, except for the first two elements.
+>**NOTE**: The Nth Fibonacci number is $F_N$ for our chosen N. By definition, $F_0 = 0$ and $F_1 = F_2 = 1$.
+> Each following value is calculated with the formula $F_N = F_{N - 1} + F_{N - 2}$.
 >
->**TIP**: For example, if the value stored in `eax` is equal to `5`, a correct solution will display `3` and for `7`, it will display `8`.
+>**TIP**: For example, if the value stored in `ecx` is equal to `5`, a correct solution will display `5` and for `7`, it will display `13`.
```

Comment:
The website doesn't support LaTeX yet. Better convert these to backticks for now.